dev/WindowsAppRuntime_BootstrapDLL/MddBootstrap.h

```diff
@@ -1,12 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
```

Comment:
>Licensed [](http://example.com/codeflow?start=3&length=8)

Do you still need `#pragma once` here?

Reply:
No. VS got smarter some years back recognizing the decades old pattern

```
#if !defined(FOO)
#define FOO
...
#endif
```

as "this file has zero compilation content don't reprocess" just like `#pragma once`.

A popular enough topic it actually has its own Wikipedia page https://en.wikipedia.org/wiki/Pragma_once